schemas/2014-04-01/Microsoft.Sql.json

```diff
@@ -84,6 +87,52 @@
       ],
       "description": "Microsoft.Sql/servers"
     },
+    "servers_administrators": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "oneOf": [
+            {
+              "type": "string"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
```

Comment:
Thanks, @TianoMS can you merge as-is and I will file a separate PR to regenerate when Azure/autorest#2401 is fixed?